### ISSUE TITLE
Tuple unpacking in return statement with parentheses causes error in Python <3.8

### ISSUE THREAD

**Issue by MattiasOlla** [microsoft/pyright#776]
**Describe the bug**
In the VS Code extension using Python 3.7 (Windows 10). When unpacking one tuple into another in a return statement (as in `test1` and `test2` below), PyRight displays an error
> Unpack operation not allowed in tuples prior to Python 3.8

regardless of whether the created tuple is wrapped in parentheses or not. In `test2`, this error is correct, [this is not allowed before Python 3.8](https://docs.python.org/3.8/whatsnew/3.8.html#other-language-changes). However, in `test1`, a tuple is created via normal tuple unpacking syntax, which [has been allowed since Python 3.5](https://www.python.org/dev/peps/pep-0448/).

**To Reproduce**
```
def test1():
    a = [1, 2, 3]
    b = (4, *a, 5)  # This shows no error, as expected
    return (4, *b, 5)  # This causes an error, incorrectly


def test2():
    a = [1, 2, 3]
    return 4, *a, 5  # Here the error is correct
```

**Expected behavior**
In `test2`, the error should be displayed for Python <3.8, but not in `test` which should be allowed for Python >=3.5.

**VS Code extension or command-line**
VS Code extension

**Additional context**
- Python 3.7.4
- PyRight VS Code extension v1.1.47
- VS Code 1.46.1
- Windows 10 Pro 1909, 64 bit

**Comment by erictraut** [comment]
Thanks for the bug report. This will be fixed in the next version of Pyright.

**Comment by erictraut** [comment]
This is now fixed in Pyright version 1.1.48, which I just published. It will also be included in the next public release of Pylance.
